### DISCUSSION TITLE
Tailwind CSS not being bundled in static export

### DISCUSSION THREAD

**Discussion by Lite5h4dow** [vercel/next.js#23195] [Help] [answered]
### What version of Next.js are you using?

10.0.8

### What version of Node.js are you using?

12.21.0

### What browser are you using?

Electron / Chromium

### What operating system are you using?

Windows / Linux

### How are you deploying your application?

next export

### Describe the Bug

Next only correctly bundles / builds Tailwind css for dev / if next js is run as a server. it appears to fail to correctly export tailwind styles in the static export with no errors.

### Expected Behavior

I would expect Tailwind CSS to work with the static export like it does in dev

### To Reproduce

Download the Next JS & Tailwind CSS example template. 
cd into directory, run `yarn next build && yarn next export` 
there are no styles on the resulting html file.

**Comment by akellbl4** [comment]
Hi there
I tried to do so and I can see CSS files in build.
Also, I tried to run server with `serve` package and everything just work.

<details>
<summary>Screenshots</summary>

<img width="332" alt="CleanShot 2021-03-18 at 22 35 57@2x" src="https://user-images.githubusercontent.com/2330682/111686698-69d54f80-883a-11eb-8ed6-ace02ad70703.png">
<img width="592" alt="CleanShot 2021-03-18 at 22 38 17@2x" src="https://user-images.githubusercontent.com/2330682/111686948-c0db2480-883a-11eb-8bdc-bc5278383e8c.png">
</details>

**Comment by Lite5h4dow** [comment, marked as answer]
this turned out to be an issue with tailwinds config. if you use the ./src/pages file tree like me then the tailwind css dosent generate properly.
get your purge looking like this ```purge: ["./src/pages/**/*.{js,ts,jsx,tsx}", "./src/components/**/*.{js,ts,jsx,tsx}"]```
